src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
extra
bc you have query like https://mate.academy/students-api//users

```suggestion
export const BASE_URL = 'https://mate.academy/students-api';
```